drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -233,8 +236,18 @@ def studio_view(self, context):
         for js_url in js_urls:
             fragment.add_javascript_url(self.runtime.local_resource_url(self, js_url))
 
+        # Do a bit of manipulation so we get the appearance of a list of zone options on
+        # items that still have just a single zone stored
+        data = copy.deepcopy(self.data)
```

Comment:
Okay, fun investigation on this one! This may be old news to you guys, but it looks like only when a function is marked as a handler (for example, via the `@XBlock.json_handler` decorator) will changes to the XBlock state be saved automatically. Because `studio_view()` is not a handler, we can mess with `self` as much as we want, and no changes will be saved to the backing store.

I'm going to remove the `copy.deepcopy` call, but I'm going to add a comment noting exactly why we can get away with that.